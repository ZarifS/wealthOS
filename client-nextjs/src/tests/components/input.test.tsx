import { render } from '@testing-library/react';
import Input from '../../components/input';

describe('Input component', () => {
  it('renders correctly', () => {
    const { asFragment } = render(
      <Input label="Email" name="email" inputType="email" onChange={() => {}} />
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
