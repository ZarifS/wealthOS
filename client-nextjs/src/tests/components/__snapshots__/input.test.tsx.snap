// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input component renders correctly 1`] = `
<DocumentFragment>
  <div
    class="main"
  >
    <label
      class="label"
      for="email"
    >
      Email
    </label>
    <input
      class="input"
      id="email"
      name="password"
      type="email"
      value=""
    />
  </div>
</DocumentFragment>
`;
